lib/chef/util/windows/net_user.rb

```diff
@@ -120,49 +91,32 @@ def initialize(username)
     @name = multi_to_wide(username)
   end
 
-  LOGON32_PROVIDER_DEFAULT = 0
-  LOGON32_LOGON_NETWORK = 3
+  LOGON32_PROVIDER_DEFAULT = Security::LOGON32_PROVIDER_DEFAULT
+  LOGON32_LOGON_NETWORK = Security::LOGON32_LOGON_NETWORK
   #XXX for an extra painful alternative, see: http://support.microsoft.com/kb/180548
   def validate_credentials(passwd)
```

Comment:
in retrospect, this should have been something like ```valid_credentials?``` but wutevs. #breakfastwhenevs